config/routes.rb

```diff
@@ -1,5 +1,20 @@
 Rails.application.routes.draw do
-  root "homes#index"
+  devise_for :users
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "monsters#index"
+  resources :monsters
```

Comment:
Are there erb rendered views for monsters?  If not we can remove this line as we will be using API endpoints for our fetch requests to interact with.